tests/mockers.py

```diff
@@ -344,8 +344,9 @@ def launch_command(*args: str) -> str:
         return out.getvalue()
 
 
-def assert_command(cmds: Iterable[str], expected_result: str, strip_indentation: bool = True) -> None:
-    assert launch_command(*cmds) == (adapt(expected_result) if strip_indentation else expected_result)
+def assert_command(cmds: Iterable[str], expected_result: str, strip_indentation: bool = True, indent: str = '  ') -> None:
```

Comment:
Is this change spurred by the original fix, or just incidental? 🤔

Reply:
@PawelLipski For some test cases the line `launch_command(*cmds) == (adapt(expected_result) if strip_indentation else expected_result)` did not work as expected so i had to change it